packages/mdc-theme/README.md

```diff
@@ -11,6 +11,21 @@ path: /catalog/theme/
 
 The Material Design color system can be used to create a color scheme that reflects your brand or style.
 
+The colors in this module are derived from three theme colors:
```

Comment:
Does background replace surface here? Or are both still used?

Reply:
Surface was introduced later, but I agree that restoring this part of the docs would be confusing. The main focus of this PR should be the five text styles.

Reply:
Gotcha - I will remove this part.